media-gfx/qview/qview-3.0.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="Minimalist image viewer"
+DESCRIPTION="qView is an image viewer designed with minimalism and usability in mind."
+HOMEPAGE="https://interversehq.com/qview/"
+COPYRIGHT="2018-2020 jurplel and qView contributors"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/jurplel/qView/archive/3.0.zip"
+CHECKSUM_SHA256="5bc2a2eb3aa6329fe6f207a72497f115625d84a0e76663872fd2e401ea7c5ae1"
+# Este parametro es para que pueda encontrar la carpeta de fuentes luego de desempaquetar el zip:
```

Comment:
I can't understand what it means, maybe leave it out?
```diff
- # Este parametro es para que pueda encontrar la carpeta de fuentes luego de desempaquetar el zip:
```

Reply:
Oops! Sorry! That was a comment for me, but I forget to delete it before send the PR.

There is any way to "accept" the change that you suggest, or I have to edit the original recipe file and commit a new version?

Reply:
I see you are using the master branch to commit the changes, if nothing else has changed it should be OK to just edit your local copy of this recipe and push the changes.
If you want to continue contributing try to see if you can use branches (and leave master branch unchanged)